config.json

```diff
@@ -496,6 +496,11 @@
       "difficulty": 1,
       "slug": "forth",
       "topics": []
+    },
+    {
+      "difficulty": 1,
+      "slug": "change",
+      "topics": []
```

Comment:
Def the first three. I see Recursion/Searching as possibilities in the solution, and as such using them as topics may steer users down a particular solution path?